src/Framework/test/SharedFxTests.cs

```diff
@@ -175,6 +175,15 @@ public void SharedFrameworkAssembliesHaveExpectedAssemblyVersions()
                 var reader = peReader.GetMetadataReader(MetadataReaderOptions.Default);
                 var assemblyDefinition = reader.GetAssemblyDefinition();
 
+                if (assemblyDefinition.GetAssemblyName().Name is
```

Comment:
@JunTaoLuo when you return, could you have a look at this change? Some of these runtime versions have been updated. It's not super clear if that's problematic (I would think not?)